desktop/dashboard.md

```diff
@@ -227,23 +227,40 @@ To interact with remote repositories:
 
 ## Explore volumes
 
-The **Volumes** view in Docker Dashboard enables you to easily create and delete [volumes](../storage/volumes.md) and see which ones are being used. If you are a developer subscribed to a Pro or a Team plan, you can also see which container is using a specific volume and explore the files and folders in your volumes.
+A [volume](../storage/volumes.md) is a directory on a disk or in a container. Volumes are directly managed by Docker and are stored as part of the host file system.
+
+The **Volumes** view in Docker Dashboard enables you to easily create and delete volumes and see which ones are being used. If you are a developer subscribed to a Pro or a Team plan, you can also see which container is using a specific volume and explore the files and folders in your volumes.
 
 {% include upgrade-cta.html
   body="To start exploring the details in your volumes, upgrade to a Pro or a Team plan."
   header-text="This feature requires a Pro or a Team plan"
   target-url="https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade_volumes"
 %}
 
+The following table lists the Volumes UI features available in each plan:
+
+{% assign yes = '![yes](/images/green-check.svg){: .inline style="height: 14px; margin: 0 auto"}' %}
+
+| Feature      | Free | Pro | Team   |
+|:-------------|:---------:|:----------:|:----------:|
+|  View, create, and delete volumes| {{ yes }}    | {{ yes }}   |  {{ yes }}    |
+| See which volumes are in use | {{ yes }} | {{ yes }}   |  {{ yes }}    |
+| See the size of the volume | {{ yes }}  | {{ yes }}   |  {{ yes }}    |
+| See volume data details (modified date, size, kind) | X | {{ yes }}   |  {{ yes }}    |
```

Comment:
Sorry, I forgot we also removed the kind column!

```suggestion
| See volume data details (modified date, size) | X | {{ yes }}   |  {{ yes }}    |
```

Reply:
Thanks Steph. Updated.